src/components/TextArea/TextArea.module.scss

```diff
@@ -1,26 +1,42 @@
+@import '../../styles/global';
+
 .textAreaContainer {
-  font-size: 1rem;
+  width: 70.4rem;
 
-  padding: 0.12rem;
+  border-radius: 1.2rem 1.2rem 0 0;
 
-  background-color: rgb(105, 53, 203);
-  border-radius: 0.3rem;
-}
+  .mainTitle {
+    width: 100%;
+
+    font-family: $mainFont;
+    color: $primaryWhite;
+    font-size: 2.2rem;
+    line-height: 2.8rem;
+
+    padding: 1.8rem 2.4rem;
+    box-sizing: border-box;
+
+    background-color: $secondaryPurple;
+    border-radius: 1.2rem 1.2rem 0 0;
+  }
+
+  .textArea {
+    width: 100%;
+    min-height: 20rem;
+    max-height: 10rem;
+    overflow-y: auto;
 
-.textArea {
-  width: 100%;
-  min-height: 4rem;
-  max-height: 10rem;
-  overflow-y: auto;
+    font-family: $mainFont;
+    color: $secondaryGray;
 
-  font-size: 1rem;
+    font-size: 1.6rem;
 
-  padding: 0.6rem;
-  border: 1px solid rgb(219, 193, 255);
-  box-sizing: border-box;
+    padding: 2.6rem 1.6rem;
+    border: 1px solid $primaryGray;
+    box-sizing: border-box;
 
-  background-color: rgb(250, 250, 250);
-  border-radius: 0.3rem;
+    background-color: rgb(250, 250, 250);
```

Comment:
Esse rgb nao tá no tema? Verificar se for um base white que a gente usa no projeto e colocar no tema.